src/tsWorker.ts

```diff
@@ -415,6 +415,29 @@ export class TypeScriptWorker implements ts.LanguageServiceHost, ITypeScriptWork
 	async updateExtraLibs(extraLibs: IExtraLibs): Promise<void> {
 		this._extraLibs = extraLibs;
 	}
+
+	async provideInlayHints(fileName: string, start: number, end: number): Promise<readonly ts.InlayHint[]> {
+		if (fileNameIsLib(fileName)) {
+			return [];
+		}
+		const preferences: ts.InlayHintsOptions = {
+			includeInlayParameterNameHints: "all"
+		};
```

Comment:
Could this be made configurable via `monaco.languages.javascriptDefaults`/`monaco.languages.typescriptDefaults`? I know we don't really have a precedent for such a deep configuration in `monaco-typescript`, right now. Speaking from my own experience, I love this feature in Visual Studio, but only for literal values where it's not obvious what the parameter is for. And especially as someone who uses the Monaco editor to teach beginner programmers, inlay hints for literal values would be awesome to have, but anything else probably overwhelming. A configuration option would be greatly appreciated 🙇‍♂️

Reply:
Yep sure. I suppose to add some options.

Reply:
https://github.com/microsoft/monaco-typescript/commit/26d78129faf0c9a6b1b8fbf848ca5ee78bcdaea2 is an example of adding a config variable to monaco-typescript

Reply:
I'm not sure should I add a new option like 'inlayHintsOptions' or just add these into some existed option?